public_wallet_app/css/styles.css

```diff
@@ -95,6 +95,15 @@ button.danger {
 button.danger:hover {
   background-color: #9c2929;
 }
+button.positive {
+  background-color: #38a14f;
+}
+button.positive:hover {
```

Comment:
we are no longer changing colors on most button hovers (expect the logout refresh buttons), just doing the slight size transform instead